README.md

```diff
@@ -115,6 +115,11 @@ _suse-rancher-setup_ uses and conforms to the [EOS Design System](https://suse.e
 
 ## Packaging
 
+In order to have the necessary info for packaging, some content should be obtained
+from secrets repo.
```

Comment:
This is specific to our team and does not apply in general. If we internally need such documentation it needs to be internal to SUSE and not in the public documentation of a public project.

Reply:
Ack